romana-install/Vagrantfile

```diff
@@ -0,0 +1,92 @@
+# -*- mode: ruby -*-

```

Comment:
Where does this file come from? Was it originally created through the template by Caleb's scripts?